src/locales/en.json

```
@@ -3,7 +3,10 @@
     "pageTitle": "RS React Redux Base"
   },
   "home": {
-    "welcomeMsg": "Welcome to React Redux Base",
+    "welcomeMsg": "Welcome to TARGET",
+    "title": "Find people near you & Connect",
+    "descriptionItem1": "Create a  target  by clicking wherever on the map, specify  the ratio and and a topic: Travel, Dating, Music, etc.",
+    "descriptionItem2": "TARGET will start a chat whenever you've a match. \nYou can always dismiss a conversation if you're not interested.",
```

if you use a new line, I think you don´t need the space

```suggestion
    "descriptionItem2": "TARGET will start a chat whenever you've a match.\nYou can always dismiss a conversation if you're not interested.",
```